cli/login.go

```diff
@@ -449,3 +553,75 @@ func openURL(inv *serpent.Invocation, urlToOpen string) error {
 
 	return browser.OpenURL(urlToOpen)
 }
+
+func promptTrialInfo(inv *serpent.Invocation, fieldName string) (string, error) {
+	value, err := cliui.Prompt(inv, cliui.PromptOptions{
+		Text: fmt.Sprintf("Please enter %s:", pretty.Sprint(cliui.DefaultStyles.Field, fieldName)),
+		Validate: func(s string) error {
+			if strings.TrimSpace(s) == "" {
+				return xerrors.Errorf("%s is required", fieldName)
+			}
+			return nil
+		},
+	})
+	if err != nil {
+		if errors.Is(err, cliui.Canceled) {
+			return "", nil
+		}
+		return "", err
+	}
+	return value, nil
+}
+
+func promptDevelopers(inv *serpent.Invocation) (string, error) {
+	options := []string{"1-100", "101-500", "501-1000", "1001-2500", "2500+"}
+	selection, err := cliui.Select(inv, cliui.SelectOptions{
+		Options:    options,
+		HideSearch: false,
+		Message:    "Select the number of developers:",
+	})
+	if err != nil {
+		return "", xerrors.Errorf("select developers: %w", err)
+	}
+	return selection, nil
+}
+
+func promptCountry(inv *serpent.Invocation) (string, error) {
+	countries := []string{
```

Comment:
this list is taken from [countries.tsx](https://github.com/coder/coder/blob/main/site/src/pages/SetupPage/countries.tsx) to have a select and filter option for this field, so the list will be maintained in two places, I'm open to options if there is a better way of handling this

Reply:
I would suggest making it a separate follow-up PR, but we could instead define countries in Go and have a separate script similar to `scripts/apitypings` to generate `countries.tsx` from `countries.go`.

Reply:
sure, will address this in a separate PR, shall I create an issue to track this ?

Reply:
I went ahead and created one: https://github.com/coder/coder/issues/15074